core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -723,46 +723,42 @@ public static String getDbVersion() {
     }
     
     public static String getDarwinAuthCheckUrl() {
-        String darwinAuthUrl;
+        String darwinAuthUrl = "";
         try{
             darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL).trim();            
         }
-        catch (NullPointerException e){
-            darwinAuthUrl = "";
-        }
+        catch (NullPointerException e){}
+        
         return darwinAuthUrl;
     }
     
     public static String getDarwinResponseUrl() {
-        String darwinResponseUrl;
+        String darwinResponseUrl = "";
         try{
             darwinResponseUrl = properties.getProperty(DARWIN_RESPONSE_URL).trim();
         }
-        catch (NullPointerException e) {
-            darwinResponseUrl = "";
-        }
+        catch (NullPointerException e) {}
+        
         return darwinResponseUrl;
     }
     
     public static String getDarwinAuthority() { 
-        String darwinAuthority;
+        String darwinAuthority = "";
         try{
             darwinAuthority = properties.getProperty(DARWIN_AUTHORITY).trim();
         }
-        catch (NullPointerException e) {
-            darwinAuthority = "";
-        }
+        catch (NullPointerException e) {}
+        
         return darwinAuthority;
     }
     
     public static String getCisUser() {
-        String cisUser;
+        String cisUser = "";
         try{
             cisUser = properties.getProperty(CIS_USER).trim();
         }
```

Comment:
@pieterlukasse I don't think it would really make sense to be loading the same properties file twice in `GlobalProperties`. 

I guess to avoid code redundancy, if this is the approach we ultimately want to take, `Properties` can be replaced with `TrimmedProperties` when loading the properties file within `GlobalProperties` but that's probably a discussion for another time with the rest of the group, or at least with @jjgao and @n1zea144. 

For now though I'll just use .trim() unless the decision to switch classes is made.